lib/testing/src/equality/equality.dart

```diff
@@ -94,7 +94,7 @@ class _EqualityGroupMatcher extends Matcher {
     });
 
     // Run the test multiple times to ensure deterministic equals
-    for (var run in range(_repetitions)) {
```

Comment:
`i` is just as unused as `run` here. Who's complaining about this and why have they never written Python?